integration/bin/run-demo-hook-service.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Usage: run-demo-hook-service.sh
+# Builds a docker image and launches the demo hook service.
+
+INTEGRATION_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
+SRC_DIR="${INTEGRATION_DIR}/src/demo_hook_service"
+NAME=demo-hook-service
+
+echo "Building docker images for ${NAME}"
+docker build -t ${NAME} ${SRC_DIR}
+
+# Expose port 5131 (used for demo hook service)
+docker create \
+       -i \
+       -t \
+       --rm \
+       -p 5131:5131 \
+       --name=$NAME \
+       ${NAME}:latest
+
+docker network connect bridge ${NAME}
+docker network connect cook_nw ${NAME}
+
+docker start -ai ${NAME}
```

Comment:
Do we need a new service? 

It seems like we could build a pretty simple dummy plugin that acts based on job labels, e.g.:

- if job has label `dummy-plugin.defer-launch-seconds` = `X`, then defer launching that job for `X` seconds
- if job has label `dummy-plugin.submission-blocking` = `on`, then reject that job submission

If we can pull this off, I think it would be a big win:

- avoids adding a new service to the local and travis setup
- avoids having to run the tests as `serial` because it's inherently job-specific

Reply:
Changed.